### ISSUE TITLE
[1.9] bundle-visualizer crashes app

### ISSUE THREAD

**Issue by faburem** [meteor/meteor#10880]
After upgrading our Meteor application to 1.9 and adding the bundle-visualizer package it crashes on start-up. It worked as expected in version 1.8.3 - both versions tested on MacOS Catalina.

**Expected behaviour:**
The application can be built without errors using the bundle-visualizer package.

**Actual behaviour:**
It does not matter if we add the bundle-visualizer package or use the --extra-packages command, it crashes either way. This does not happen in a new Meteor project, therefore it has to be related to one or more packages we are using but we could not figure it out in more detail yet.
Here is a quick reproduction scenario:
1) clone our app repo https://github.com/kromitgmbh/titra
2) in the app directory run `meteor run --production --extra-packages bundle-visualizer`

```
=> Errors prevented startup:                  
   While minifying app code:
   ~/.meteor/packages/standard-minifier-js/.2.6.0.1cijq55.rpvn++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/location.js:41:63:
   Unexpected token (1:8) while minifying dynamic/node_modules/meteor/bundle-visualizer/style.css
```
Thanks for your help!

**Comment by afrokick** [comment]
Did you try to update `standard-minifier-js` package?

**Comment by faburem** [comment]
We are using the latest version:
<img width="531" alt="Screenshot 2020-01-22 at 08 42 04" src="https://user-images.githubusercontent.com/11456790/72874829-3446dc00-3cf3-11ea-8c62-24bceed3e51b.png">

**Comment by DavidSichau** [comment]
We can confirm this issue in one of our apps.

Here our error log:

```
=> Errors prevented startup:
   
   While minifying app code:

   /home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/location.js:41:63:
   Unexpected token (1:8) while minifying
   dynamic/node_modules/meteor/bundle-visualizer/style.css
   at Object.raise
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/location.js:41:63)
   at Object.unexpected
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/util.js:160:16)
   at Object.parseIdentifierName
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/expression.js:2160:18)
   at Object.parseIdentifier
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/expression.js:2127:23)
   at Object.parseFunctionId
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:1095:52)
   at Object.parseFunction
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:1047:22)
   at Object.parseFunctionStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:555:17)
   at Object.parseStatementContent
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:185:21)
   at Object.parseStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:141:17)
   at Object.parseStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/plugins/flow.js:1675:28)
   at Object.parseBlockOrModuleBlockBody
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:860:25)
   at Object.parseBlockBody
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:836:10)
   at Object.parseTopLevel
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:47:10)
   at Object.parseTopLevel
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/plugins/flow.js:2913:30)
   at Object.parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/index.js:43:10)
   at Object.parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/index.js:58:38)
   at parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/parser.js:7:22)
   at Object.minify
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:184:5)
   at Object.minify (packages/babel-compiler.js:59:29)
   at meteorJsMinify (packages/minifier-js.js:60:25)
   at packages/minifyStdJS/plugin/minify-js.js:126:20
   at Array.forEach (<anonymous>)
   at MeteorBabelMinifier.processFilesForBundle
   (packages/minifyStdJS/plugin/minify-js.js:118:9)
   
   
   While minifying app code:

   /home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/location.js:41:63:
   Unexpected token (1:8) while minifying
   dynamic/node_modules/meteor/bundle-visualizer/style.css
   at Object.raise
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/location.js:41:63)
   at Object.unexpected
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/util.js:160:16)
   at Object.parseIdentifierName
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/expression.js:2160:18)
   at Object.parseIdentifier
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/expression.js:2127:23)
   at Object.parseFunctionId
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:1095:52)
   at Object.parseFunction
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:1047:22)
   at Object.parseFunctionStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:555:17)
   at Object.parseStatementContent
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:185:21)
   at Object.parseStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:141:17)
   at Object.parseStatement
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/plugins/flow.js:1675:28)
   at Object.parseBlockOrModuleBlockBody
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:860:25)
   at Object.parseBlockBody
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:836:10)
   at Object.parseTopLevel
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/statement.js:47:10)
   at Object.parseTopLevel
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/plugins/flow.js:2913:30)
   at Object.parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/parser/index.js:43:10)
   at Object.parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/@babel/parser/src/index.js:58:38)
   at parse
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/parser.js:7:22)
   at Object.minify
   (/home/sichaud/.meteor/packages/standard-minifier-js/.2.6.0.olpsgw.1xyma++os+web.browser+web.browser.legacy+web.cordova/plugin.minifyStdJS.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:184:5)
   at Object.minify (packages/babel-compiler.js:59:29)
   at meteorJsMinify (packages/minifier-js.js:60:25)
   at packages/minifyStdJS/plugin/minify-js.js:126:20
   at Array.forEach (<anonymous>)
   at MeteorBabelMinifier.processFilesForBundle
   (packages/minifyStdJS/plugin/minify-js.js:118:9)
   
   
=> Your application has errors. Waiting for file change.
```

**Comment by faburem** [comment]
We get exactly the same error log.
@DavidSichau: which packages are you using?
Below you can find ours, maybe we can figure out a common ground and isolate a package causing it:
<img width="931" alt="Screenshot 2020-01-24 at 11 06 02" src="https://user-images.githubusercontent.com/11456790/73061118-b3761480-3e99-11ea-8add-e665f5c277f9.png">

**Comment by fafournier** [comment]
I got the same issue ! So I got looking.

The error actually is shown in the stack trace : dynamic/node_modules/meteor/bundle-visualizer/style.css,  CSSlint.net tells me that there's a trailing comma that is messing up the minification near line 41. It's actually on line 40 and when I remove it, it works on my workspace (using a local copy of the bundle-visualizer package). I'll try to do a pull request as soon as I understand the contributing guidelines...
